alembic/versions/77856f165be7_bug_1403172_fix_fennecandroid_api_16.py

```diff
@@ -0,0 +1,86 @@
+"""bug 1403172 fix fennecandroid api 16
+
+Revision ID: 77856f165be7
+Revises: a38b03765a79
+Create Date: 2017-09-28 20:41:56.221487
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '77856f165be7'
+down_revision = 'a38b03765a79'
+
+from alembic import op
+from socorro.lib import citexttype, jsontype, buildtype
+from socorro.lib.migrations import fix_permissions, load_stored_proc
+
+import sqlalchemy as sa
+from sqlalchemy import types
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import table, column
+
+
+def upgrade():
+    op.execute("""
+    INSERT INTO release_repositories VALUES ('mozilla-central-android-api-16');
+    INSERT INTO release_repositories VALUES ('mozilla-aurora-android-api-16');
+    INSERT INTO release_repositories VALUES ('mozilla-beta-android-api-16');
+    INSERT INTO release_repositories VALUES ('mozilla-release-android-api-16');
```

Comment:
I'm mimicking what @rhelmer did in https://github.com/mozilla-services/socorro/blob/master/alembic/versions/335c2bfd99a6_bug_1255444_fix_fennecandroid_for_new_.py#L27

But I don't know whether we need all of those or not.

In this directory, we see the last of the mozilla-central-android-api-15 on 2017-08-29:

http://archive.mozilla.org/pub/mobile/nightly/2017/08/

And you see the first of the mozilla-central-android-api-16 on 2017-08-30.

I don't see the other release repositories used anywhere. Do we need to add them?

Reply:
I'm not sure if including aurora make sense post-[Dawn](http://release.mozilla.org/firefox/release/2017/04/17/Dawn-Project-FAQ.html).

Reply:
I talked to @rhelmer who is super busy and was able to answer my question.

"Yes", we need them except "aurora". The value is two parts squished together where the first part is the repository and the second is the platform.

I'll update the PR accordingly and move this forward.